content/en/docs/tasks/debug/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Debugging"
```

Comment:
Maybe we should keep the current title "Monitoring, Logging, and Debugging" so that users can still quickly find the topic they want.